cassandra-analytics-core/src/main/java/org/apache/cassandra/spark/sparksql/CassandraDataSink.java

```diff
@@ -73,7 +71,7 @@ public BaseRelation createRelation(@NotNull SQLContext sqlContext,
             case Append:
                 // Initialize the job group ID for later use if we need to cancel the job
                 // TODO: Can we get a more descriptive "description" in here from the end user somehow?
-                BulkWriterContext writerContext = createBulkWriterContext(
+                BulkWriterContext writerContext = factory().createBulkWriterContext(
```

Comment:
Realize it didn't happen in this PR, but the comments above this line has somehow never been kept with the code that they actually apply to - can you please move them back to the line where we actually call `setJobGroup` and remove the TODO at this point?